public/js/script.js

```diff
@@ -223,6 +224,62 @@ $('textarea').keyup(function(e) {
           });
     }
 
+    else if(command.split(" ")[0].trim()=="rm -r")
+    {
+        if(command.split(" ").length == 1) {
+        $('.terminal-output').append('<div class="command" role="presentation" aria-hidden="true"><div style="width: 100%;"><span class="user">root@' + username + ': ~$ </span><span>' + command + '</span></div></div>');
+        $('.terminal-output').append('<div class="result"><div style="width: 100%;"><span>rm -r: missing operand &ltfolder name&gt </span></div></div><br>');
+        reset();
+        return;
+      }
+      $.ajax({
+        type:'get',
+        datatype :'json',
+        data:{nameFolder: command.split(" ")[1].trim()},
```

Comment:
If the user enters `rm -r folder` then `command.split(" ")[1]` would give `-r`.

Reply:
@namrata013  Your nameFolder is `-r` because of this line that is why it is not deleting. You have to take `command.split(" ")[2].trim()` for it to work. Also you need to pass directory otherwise you will not be able to delete when you're inside a folder.